mne/beamformer/_lcmv.py

```diff
@@ -285,6 +330,12 @@ def lcmv(evoked, forward, noise_cov, data_cov, reg=0.05, label=None,
         detected automatically. If int, the rank is specified for the MEG
         channels. A dictionary with entries 'eeg' and/or 'meg' can be used
         to specify the rank for each modality.
+    whitening: False | True
+        whiten leadfield / covariance matrix
+    weight_norm: None | 'nai' | 'unitnoisegain'    
+        If 'nai', the Neural Activity Index (Van Veen, 1997) will be computed,
+        if 'unitnoisegain', the unit-noise gain minimum variance beamformer 
+        will be computed (Borgiotti-Kaplan beamformer)
```

Comment:
The current code uses unitnoisegain ? If so then the default should be unitnoisegain ant not None right?